### DISCUSSION TITLE
Super Mario 3D All-Stars

### DISCUSSION THREAD

**Discussion by breakbreakdown84** [skyline-emu/skyline#650] [Game Requests]
Super Mario 3D All-Stars

**Comment by ShadowBlakc** [comment]
En el futuro ire por sus almas y las de sus familiares por corromper mas mi
paz mental...

Que el skyline solo será para celulares de gama alta?!,
👿 Me vengare

**Comment by mjcox244** [comment]
This most likely wont happen for a while, SM64 needs the switch's JIT service, sushine is a gamecube emulator with a texturepack and a slightly modded ISO, and galaxy is 32bit with gpu emulation.

**Reply by MysticAx0lotl** [reply]
SM64 and Sunshine can already be emulated on Mupen64 and Dolphin respectively with _much better_ results than SM3DAS. 1080p (or even 4K), wide-screen on both games (with some tinkering), texture packs, and mods (like Super Mario Sunburn). The only game I'd see a legitimate reason to get working on Skyline is Super Mario Galaxy, because of its touch friendly controls (and it doesn't perform too well on Android Dolphin, at least not on my phone)
